src/content/blog/new-domain-new-discord-same-project.md

```diff
@@ -0,0 +1,60 @@
+---
+title: New Domain, new Discord, same Project!
+description: .xyz is out, .dev is the new black.
+publishDate: 2024-11-27
+ogVariant: winter
+author: louis-escher
+---
+
+Hey all!
+
+Although it's been a while since we last talked, I can assure you that work on StudioCMS has been moving forward. We've been making good progress on 
+`beta.8` and hope to release it towards the end of 2024, although that's more of a guess than a prediction.
+
+You've read the title (or so I hope), so what am I on about here?
+
+Well, you've no doubt seen that we have been [sponsored by Turso](https://studiocms.dev/blog/turso-sponsorship), which were and still are incredibly 
+grateful for. However, we're trying to improve our experience as much as possible for both new users and contributors to the project. At this point 
+we feel we can move away from Astrolicious, the community organization who's been helping us nurture our little project, and go in alone to have full 
+control over our own ecosystem! After talking to [Florian](https://github.com/florian-lefebvre) and [Alex](https://github.com/alexanderniebuhr), 
+we've come to the conclusion that it's time for a fresh coat of new paint. Therefore, I introduce to you:
+
+## The new StudioCMS Discord Server
```

Comment:
```suggestion
## The new StudioCMS Discord server
```

Reply:
As this is a heading, according to the Chicago Manual of Style "Server" can have a capital S here, I'd prefer to leave it that way